src/workspace.coffee

```diff
@@ -371,9 +371,11 @@ class Workspace extends Model
   Section: Opening
   ###
 
-  # Essential: Open a given a URI in Atom asynchronously.
-  #
-  # * `uri` A {String} containing a URI.
+  # Essential: Open a given URI in Atom asynchronously. 
+  # If no URI is given, or URI does not resolve to an existing file, 
```

Comment:
Perhaps instead of saying:

> or URI does not resolve to an existing file,

we could say something like

> or URI is not already open in an existing pane item

because URIs are used to reference and open more than files. For example, `atom.workspace.open("atom://config")` can be used to open the Settings View. What do you think, @vegar?